full-service/src/db/models.rs

```diff
@@ -104,6 +105,44 @@ pub struct Account {
     pub fog_enabled: bool,
 }
 
+/// A View Only Account entity.
+///
+/// Contains the account view private key
+#[derive(Clone, Serialize, Identifiable, Queryable, PartialEq, Debug)]
+#[primary_key(id)]
+pub struct ViewOnlyAccount {
+    /// Primary key
+    pub id: i32,
+    /// An additional ID, derived from the account data.
+    pub account_id_hex: String,
+    /// private key for viewing MobileCoin belonging to an account.
+    pub view_private_key: Vec<u8>,
+    /// Index of the first block where this account may have held funds.
+    pub first_block_index: i64,
```

Comment:
each of the `i64` references should be `u64` where negative values don't have meaning

Reply:
I wasn't sure about this. I followed the example of all the other models in our schema, which all use i64. I agree that it would make more sense to use u64 but I think there is something with diesel that makes us use i64